src/components/profile/header.jsx

```diff
@@ -0,0 +1,29 @@
+import Image from "next/image";
+import { IoSettingsSharp } from "react-icons/io5";
+import { FaStar } from "react-icons/fa";
+
+const ProfileHeader = ({ picture, username, name, email, stars }) => {
+  return (
+    <div className="bg-code-darkgray rounded w-80 h-60 flex flex-col justify-center items-center relative font-semibold">
+      <div className="pb-3.5 flex flex-row">
+        <Image
+          className="bg-code-white rounded-full w-20 h-20"
+          src={picture}
+          alt={name + "'s Profile Picture"}
+        />
+        <div className="absolute top-2.5 right-5">
```

Comment:
this div doesn't serve a purpose, u can combine it with the settings icon directly